### PR TITLE
Bump securedrop-grsec-focal metapackage to 5.4.88

### LINKED ISSUE #5479: Provide a 5.4-series grsecurity-patched kernel with NUC10 support
## Description

Intel NUCs are the default recommendation for SecureDrop servers, but the 10-series NUCs' Comet Lake chipset is only supported by Linux 5.5+. Ubuntu 20.04 ships with a 5.4 kernel with backported support for Comet Lake.

GRSecurity provides a patch for 5.4-series kernels. In order to be able to update hardware recommendations and prepare for the transition to Focal, we should build and test a 5.4-series kernel and determine what patches are required to get the necessary NUC functionality working.

## User Research Evidence

User anecdotes about NUC availability.

### PR DESCRIPTION
**Pull request by emkll**
## Status

Ready for review
## Description of Changes

Fixes #5479 and towards https://github.com/freedomofpress/securedrop/issues/4768

Updates the kernel metapackage version to 5.4.88 for Focal installs.

See https://github.com/freedomofpress/securedrop-dev-packages-lfs/pull/89 

## Testing

- [ ] Focal metapackage pulls in 5.4.88 kernel image
- [ ] Xenial metapackage pulls in 4.14.188 kernel image
## Deployment

For new and existing installs, these changes will be distributed via deb packages through apt.

### If you made changes to the server application code:

- [ ] Linting (`make lint`) and tests (`make test`) pass in the development container

### If you made changes to `securedrop-admin`:

- [ ] Linting and tests (`make -C admin test`) pass in the admin development container

### If you made changes to the system configuration:

- [ ] [Configuration tests](https://docs.securedrop.org/en/latest/development/testing_configuration_tests.html) pass

### If you added or removed a file deployed with the application:

- [ ] I have updated AppArmor rules to include the change

### If you made non-trivial code changes:

- [x] I have written a test plan and validated it for this PR

### CODE REVIEW

File: molecule/testinfra/common/test_grsecurity.py
Comment (conorsch) [inline molecule/testinfra/common/test_grsecurity.py]: `host` isn't available outside function scope, will need a refactor.

**Review by kushaldas — DISMISSED** [review]
I have updated the test code to make sure it goes through CI.

Also verified that: 

- [x] Focal metapackage pulls in 5.4.88 kernel image
- [x] Xenial metapackage pulls in 4.14.188 kernel image

But, the `IRDA` tests are failing for the kernel: https://app.circleci.com/pipelines/github/freedomofpress/securedrop/1812/workflows/411763ec-cf39-43d3-a77b-ab89388a6b36/jobs/49865 @emkll what do you think?

**Comment by emkll** [comment]
There's currently an issue with the split logic for the config and grsec metapackages (introduced in #5684 and #5691 respectively do not produce the expected package contents due to filepath not being the same as the package name, resulting in some unwanted subfolders within the package:

```
total 36K
drwxr-xr-x  5 user    user    4.0K Feb  5 14:57 .
drwxrwxrwt 29 root root  20K Feb  8 09:56 ..
drwxr-xr-x  3 user    user    4.0K Feb  5 14:57 etc
drwxr-xr-x  3 user    user    4.0K Feb  5 14:57 -focal
drwxr-xr-x  3 user    user    4.0K Feb  5 14:57 opt
```

**Comment by conorsch** [comment]
Looks like that extraneous dir only appears in the Focal metapackage, not in the Xenial one. It's empty, but should certainly be removed. Digging a bit now.

**Comment by conorsch** [comment]
@emkll I pushed a fix that appears to resolve for both Focal & Xenial.

**Comment by conorsch** [comment]
@kushaldas This looks good to my eye, any concerns with merging?

**Comment by rmol** [comment]
I have an alternative that simplifies the distribution branching that I've been hoping to share for review. It's just been hard to test because of all the other Focal problems on develop right now. Fingers crossed it'll be ready for a look by midday, and obviously I think it's worth trying to dig out from under this a bit now, but if you'd rather just cross this one off and move on, please go ahead and merge, and maybe we can clean it up later, in any event when we remove the Xenial logic.

**Comment by conorsch** [comment]
Sounds great, @rmol, I can review later today. Happy to wait until then. Regarding keeping the distribution branching clean, I had similar thoughts on https://github.com/freedomofpress/securedrop/pull/5780, so perhaps we can reuse the approach you propose in a few places.

**Comment by kushaldas** [comment]
> @kushaldas This looks good to my eye, any concerns with merging?

Nope, I think it is good to merge when you want to.

**Comment by conorsch** [comment]
> if you'd rather just cross this one off and move on, please go ahead and merge, and maybe we can clean it up later

Indeed, some folks are blocked on Focal hardware installs, so proceeding with merge. Happy to revisit any additional logic you've got queued up, @rmol, just point me to a branch!

**Review by conorsch — APPROVED** [review]

**Pull request merged**
